sim/core/utils.go

```diff
@@ -158,6 +158,11 @@ func Flatten[T any](src [][]T) []T {
 	return dst
 }
 
+func MasteryRatingToMasteryPoints(masteryRating float64) float64 {
+	// TODO: check mastery point rounding behavior
+	return math.Floor(masteryRating / MasteryRatingPerMasteryPoint)
```

Comment:
Is this floor supposed to be here? If that is true then a character gaining 178 mastery rating could mean they get nothing because of rounding and I highly doubt the system works like that. This is close to a whole item worth of mastery being ignored

Reply:
I would suggest you remove the flooring for now as we've seen numerous times that stats are not rounded in any way. (Even the expertise in wotlk was never rounded/truncated as people used to believe)